sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -35,4 +28,23 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_burstModeSet</EFDB_Topic>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_continuousModeSet</EFDB_Topic>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_burstCountSet</EFDB_Topic>
+    <item>
+      <EFDB_Name>count</EFDB_Name>
+      <Description>The new burst count.</Description>
+      <IDL_Type>double</IDL_Type>
```

Comment:
Should be a long, to match the command? Can the laser report a meaningfully non-integral burst count (as opposed to representing an integer as a float, in which case you may as well round it before reporting it).